corehq/apps/domainsync/management/commands/copy_domain.py

```diff
@@ -330,13 +329,7 @@ def copy_doc(doc, count, sourcedb, target_couch, exclude_types, total, simulate,
               (doc["doc_type"], count, total, doc["doc_type"], doc["_id"]))
     else:
         if not simulate:
-            for i in reversed(list(range(5))):
-                try:
-                    dt = DocumentTransform(doc, sourcedb, exclude_attachments)
-                    break
-                except RequestError:
-                    if i == 0:
-                        raise
+            dt = DocumentTransform(doc, sourcedb, exclude_attachments)
```

Comment:
Can `DocumentTransform` be completely removed? Looks like it just does some stuff with attachments -> external_blobs

Reply:
Probably, but I'm going to skip doing that for now to avoid scope creep